Meridian59/Data/Models/AdminData/AdminData.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.ComponentModel;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Meridian59.Common.Interfaces;
+using Meridian59.Data.Lists;
+using Meridian59.Protocol.Events;
+using Meridian59.Protocol.GameMessages;
+
+namespace Meridian59.Data.Models.AdminData
+{
+    public class AdminData : INotifyPropertyChanged, IClearable
+    {
+        //Handles the objects we're watching in the AdminUI
+        private BaseList<AdminObject> TrackedObjects { get; set; }
+        public void TrackAdminObject(AdminObject adminObject)
+        {
+            TrackedObjects.Add(adminObject);
+            OnTrackedAdminObjectAdded(new AddTrackedAdminObjectEventHandlerArgs(this, adminObject));
+        }
+        public void UntrackAdminObject(AdminObject adminObject)
+        {
+            TrackedObjects.Remove(adminObject);
+        }
+
+        public AdminData()
+        {
+            TrackedObjects = new BaseList<AdminObject>();
+        }
+
+        /// <summary>
+        /// Interface to the Client's Network
+        /// </summary>
+        public event GameMessageEventHandler PacketSend;
```

Comment:
I'm not sure about this, construct/sending game-messages from model layer.
At least this would be the first model class in Data.Models which actually tries to send a gamemessage.